tests/handles.test.ts

```diff
@@ -0,0 +1,126 @@
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('Set User Handle', () => {
+  test('Logged In User Can Successfully Set Their Own Handle', async () => {
+    // Setup Required Data
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+
+    // Persist to Database
+    await new PortalState().createUsers(member).write();
+
+    // Get API Controller
+    const userController = ControllerFactory.user(conn);
+    const handle = 'mycustomhandle';
+
+    // Get Updated User Profile
+    const patchedUserResponse = await userController.patchCurrentUser({ user: { handle } }, member);
+
+    expect(patchedUserResponse.error).toBeNull();
+
+    const patchedUser = patchedUserResponse.user;
+
+    expect(patchedUser).toEqual({ ...member.getFullUserProfile(), handle });
+  });
+
+  test('User Cannot Set Handle Using an Invalid Handle Format', async () => {
+    // TODO: I can't get this test to reject with an error because validators aren't run here for some reason
+    // const conn = await DatabaseConnection.get();
+    // const member = UserFactory.fake();
+
+    // const shortHandle = 'a';
+    // const longHandle = 'a'.repeat(40);
+    // const specialCharHandle = '@!#!*)$#@&SAD}{ ';
+
+    // await new PortalState().createUsers(member).write();
+
+    // const userController = ControllerFactory.user(conn);
+
+    // await expect(userController.patchCurrentUser({ user: { handle: shortHandle } }, member)).rejects.toThrow('');
+    // await expect(userController.patchCurrentUser({ user: { handle: longHandle } }, member)).rejects.toThrow('');
+    // await expect(userController
+    //   .patchCurrentUser({ user: { handle: specialCharHandle } }, member)).rejects.toThrow('');
+  });
+
+  // TODO: Do we need this test if there is no UUID being passed in?
+  test('Logged In User Cannot Set a Handle for a Different User', async () => {
```

Comment:
I'm a bit unsure what this means, but it seems like this and the next test are essentially testing the same thing, so I don't think we need this test unless there's something I'm missing